hypha.php

```diff
@@ -420,7 +418,9 @@ function data($name) {
 		switch ($name) {
 			//START_OF_DATA
 			case 'index': return '';
-			case 'system/core/language.php': return "json_decode('{\"en\": \"English\"}', true);";
+			case 'system/core/language.php':
+        require_once('system/core/language.php');
+        return $isoLangList;
```

Comment:
Shouldn't this have a case for `languages.json` rather than `language.php` now? And that could just return the `file_get_contents()` for the existing `languages.json`?

Reply:
This is partly resolved, but I'll add a new comment for what's left.